sympy/__init__.py

```diff
@@ -10,6 +10,15 @@
 """
 
 from __future__ import absolute_import, print_function
+del absolute_import, print_function
+
+try:
+    import mpmath
+except ImportError:
+    raise ImportError("SymPy now depends on mpmath as an external library. "
+    "You can install mpmath with 'pip install mpmath'.")
```

Comment:
How about: "You can install mpmath with 'pip install mpmath' or see the SymPy documentation for other options."